DailyQuest/DailyQuest/Data/PersistentStorages/BrowseQuestsStorage/RealmStorage/RealmBrowseQuestsStorage.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  RealmBrowseQuestsStorage.swift
+//  DailyQuest
+//
+//  Created by 이전희 on 2022/11/15.
+//
+
+import RxSwift
+
+final class RealmBrowseQuestsStorage {
+
+    private let realmStorage: RealmStorage
+
+    init(realmStorage: RealmStorage = RealmStorage.shared) {
+        self.realmStorage = realmStorage
+    }
+}
+
+extension RealmBrowseQuestsStorage: BrowseQuestsStorage {
+    func fetchBrowseQuests() -> Observable<[BrowseQuest]> {
+        return Observable<[BrowseQuest]>.create { [weak self] observer in
+            guard let realmStorage = self?.realmStorage else {
+                return Disposables.create()
```

Comment:
assertion 에러를 띄워주면 좋을 것 같습니다. 비록 앱이 죽지만, 개발하는 동안에는 확실한 indicate를 받을 수 있을 것 같아요 :)
```suggestion
guard let realmStorage = self?.realmStorage else {
    assertionFailure("realm storage is not exist")
    return Disposables.create()
```